pkgs/linux-jovian/default.nix

```diff
@@ -109,6 +109,19 @@ buildLinux (args // rec {
     owner = "Jovian-Experiments";
     repo = "linux";
     rev = version;
-    hash = "sha256-ypYhz1kD+enIl31yhjlzqDnd3RqQcyc+7Udlw1Y5iSM=";
+    hash = "sha256-nwog+yXv9v1vENLELwJrtizRv2w7PVWivc+NvAL+Gn0=";
+
+    # Sometimes the vendor doesn't update the EXTRAVERSION tag.
+    # Let's fix it up in post.
+    # ¯\_(ツ)_/¯
+    # Also, `postPatch` on the kernel doesn't compose in `buildLinux`.
+    # ¯\_(ツ)_/¯
+    postFetch = ''
+      (
+      echo ":: Fixing-up EXTRAVERSION with actual tag"
+      cd $out
+      sed -i -e 's/^EXTRAVERSION =.*/EXTRAVERSION = -${vendorVersion}/g' Makefile
+      )
+    '';
```

Comment:
This mess is thanks to the vendor not updating `EXTRAVERSION` with this particular tag update.

I preferred making our version match the actual tag rather than match the vendor source.

This can be kept going forward as either it'll be a no-op (whitespace notwithstanding) or will fix issues that *anyway* our build tooling will be angry at.

Namely, without doing *some other acrobatics*, we need to have the package version match the "mod dir", which in turn defaults to `${version}`, which includes `vendorVersion`.